packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -324,19 +324,39 @@ export function getInternalReactConstants(
     PROFILER_SYMBOL_STRING,
     SCOPE_NUMBER,
     SCOPE_SYMBOL_STRING,
+    FORWARD_REF_NUMBER,
+    FORWARD_REF_SYMBOL_STRING,
+    MEMO_NUMBER,
+    MEMO_SYMBOL_STRING,
   } = ReactSymbols;
 
+  function resolveFiberType(type: any) {
+    // This is to support lazy components with a Promise as the type.
+    // see https://github.com/facebook/react/pull/13397
+    if (typeof type.then === 'function') {
+      return type._reactResult;
+    }
```

Comment:
I couldn't find any reference to `_reactResult` in the codebase. Is still relevant? Is it safe to remove?

Reply:
I think it has sense been renamed to `_result` now that you mention it.

Someone can follow up with that in a separate PR.

cc @acdlite

Reply:
I don't think this codepath is even used by DevTools anymore. By the time React DevTools gets called for the lazy component, it just gets the e.g. function.